virtual-classroom/src/main/java/ro/utcn/sd/mid/assign1/virtualclassroom/repository/jpa/HibernateAnswerVoteRepository.java

```diff
@@ -0,0 +1,68 @@
+package ro.utcn.sd.mid.assign1.virtualclassroom.repository.jpa;
+
+import ro.utcn.sd.mid.assign1.virtualclassroom.entity.Answer;
+import ro.utcn.sd.mid.assign1.virtualclassroom.entity.AnswerVote;
+import ro.utcn.sd.mid.assign1.virtualclassroom.repository.AnswerVoteRepository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+import java.util.List;
+import java.util.Optional;
+
+
+public class HibernateAnswerVoteRepository extends HibernateAbstractRepository<AnswerVote>
+        implements AnswerVoteRepository {
+    private final EntityManager entityManager;
+
+    public HibernateAnswerVoteRepository(EntityManager entityManager) {
+        super(entityManager, AnswerVote.class);
+        this.entityManager = entityManager;
+    }
+
+    // DOES NOT WORK, IT ADDS THE VOTE TO ALL ANSWERS
```

Comment:
Obsolete code and commented code are basically the same thing. Check out my previous comment.